carberretta/bot/cogs/mod.py

```diff
@@ -80,17 +92,153 @@ async def nickname_valid(self, nickname: str) -> bool:
             else False
         )
 
+    async def profanity_filter(self, message: discord.Message) -> None:
+        ctx = await self.bot.get_context(message, cls=commands.Context)
+
+        if ctx.command is None:
+            filter_result_raw = checkMessageList(message.content)
+            filter_result = {
+                'found': [],
+                'count': []
+            } # type: dict
+
+            if filter_result_raw:
+                await message.delete()
+
+                for word in filter_result_raw:
+                    filter_result['found'].append(word['word'] + '\n')
+                    filter_result['count'].append(str(word['count']) + '\n')
+
+                warning_msg = await message.channel.send(f"{message.author.mention} please do not use offensive language!")
+
+                member = self.bot.guild.get_member(message.author.id)
+
+                await self.modlog_channel.send(
+                    embed=discord.Embed.from_dict(
+                        {
+                            "title": "Filtered Message",
+                            "color": 0xe33838,
+                            "thumbnail": {"url": f"{member.avatar_url}"},
+                            "footer": {"text": f"ID: {message.id}"},
+                            "image": {"url": att[0].url if len((att := message.attachments)) else None},
```

Comment:
You don't need the `len(...)` call here.
```suggestion
                            "image": {"url": att[0].url if (att := message.attachments) else None},
```

Reply:
Fixed where it was copied from your code above as well